assembl/models/preferences.py

```diff
@@ -740,6 +740,52 @@ def get_preference_data_list(self):
             "item_default": ""
         },
 
+        # Allow whitelist to be applied to SSO login process
+        {
+            "id": "whitelist_on_authentication_backend",
+            "name": _("Whitelist on authentication service"),
+            "value_type": "bool",
+            # "scalar_values": {value: "label"},
+            "description": _(
+                "Allow white list to be applied to the authentication service chosen"),
+            "allow_user_override": None,
+            "modification_permission": P_ADMIN_DISC,
+            # "frontend_validator_function": func_name...?,
+            # "backend_validator_function": func_name...?,
+            "default": False
+        },
+
+        # Allow whitelist to be applied to the regular login process
+        {
+            "id": "whitelist_on_login",
+            "name": _("Whitelist on default login"),
+            "value_type": "bool",
+            # "scalar_values": {value: "label"},
+            "description": _(
+                "Allow white list to be applied to the default login process"),
+            "allow_user_override": None,
+            "modification_permission": P_ADMIN_DISC,
+            # "frontend_validator_function": func_name...?,
+            # "backend_validator_function": func_name...?,
+            "default": False
```

Comment:
are you sure default should be False?

Reply:
This should be false. We don't want to check whitelist by default.

Reply:
Btw, I did not insist, it's your choice, but an empty whitelist is not checked.
I'm not sure in what case you'd want to populate whitelist and leave both those prefs at their default of false, because it adds an operation, and hence an opportunity for making a mistake.
The most common case (with non-empty whitelist) will be that "whitelist_on_register" will be True, and "whitelist_on_authentication_backend" will be False, and this common case does not interfere with normal operations if the whitelist is left empty.

Reply:
I see your point. This is a preference, really. But, I'll side with you, as I don't trust administrators to fully know what they're doing. 
This commit should resolve this check now (it didn't before, as you had said). 011c525e